cmd/notary/cert.go

```diff
@@ -63,29 +64,30 @@ func certRemove(cmd *cobra.Command, args []string) {
 	}
 
 	var certsToRemove []*x509.Certificate
+	var certFoundByID *x509.Certificate
 
 	// If there is no GUN, we expect a cert ID
 	if certRemoveGUN == "" {
 		certID := args[0]
 		// This is an invalid ID
-		if len(certID) != idSize {
+		if len(certID) != notary.IDSize {
```

Comment:
Maybe we can do this check _if_ we fail to find the key and print it as a warning message?